src/back/utils/validations.ts

```diff
@@ -154,3 +158,27 @@ export function validateEventTypesFilters(req: Request) {
 
   return parsedEventTypes.data
 }
+
+export function validateIsDaoCommittee(user: string) {
+  if (!isDAOCommittee(user)) {
+    throw new RequestError('Only DAO committee members can update a proposal status', RequestError.Forbidden)
```

Comment:
If this is a generic validation then I'd change this to `Only DAO committee members can perform this action`. If this is only used in that proposal route then I'd move this function back there.